unittests/protocol_feature_tests.cpp

```diff
@@ -607,4 +607,101 @@ BOOST_AUTO_TEST_CASE( disallow_empty_producer_schedule_test ) { try {
 
 } FC_LOG_AND_RETHROW() }
 
+BOOST_AUTO_TEST_CASE( only_bill_to_first_authorizer ) { try {
+   tester chain( setup_policy::preactivate_feature_and_new_bios );
+
+   const auto& tester_account = N(tester);
+   const auto& tester_account2 = N(tester2);
+
+   chain.produce_blocks();
+   chain.create_account(tester_account);
+   chain.create_account(tester_account2);
+
+   resource_limits_manager& mgr = chain.control->get_mutable_resource_limits_manager();
```

Comment:
Change this to: `const resource_limits_manager& mgr = chain.control->get_resource_limits_manager();`.